html/js/loklak.js

```diff
@@ -21,3 +21,68 @@ app.controller("search", function($scope, $http) {
     }
   }
 });
+
+app.controller("topmenu", function($scope, $http) {
+  $http.get("/cms/topmenu.json").
+    success(function(data, status, headers, config){
+        $scope.results = data.items;
+    });
+    console.log($scope);
+});
+
+app.filter("reverse", function() {
+  return function(items) {
+    if (!items || !items.length) {
+      return;
+    }
+    return items.slice().reverse();
+  };
+});
+
+angular.module("loklak")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["angular" is not defined.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2623295948/issues/source?bid=3288751&fileBranchId=3312976#l42)